public/app/features/browse-dashboards/components/utils.ts

```diff
@@ -1,5 +1,15 @@
+import { config } from '@grafana/runtime';
+
 import { DashboardViewItemWithUIItems } from '../types';
 
 export function makeRowID(baseId: string, item: DashboardViewItemWithUIItems) {
   return baseId + item.uid;
 }
+
+export function isSharedWithMe(item: DashboardViewItemWithUIItems | string) {
+  if (typeof item === 'string') {
+    return item === config.sharedWithMeFolderUID;
+  }
+
+  return item.kind === 'folder' && item.uid === config.sharedWithMeFolderUID;
```

Comment:
i don't think we should support passing 2 completely different things to this function. can we just pick one? either it accepts the whole item, or it just accepts the uid

(i'm leaning towards just accepting uid)

Reply:
I agree.